woocommerce/woocommerce
# Move usePreventLeavingPage to navigation package

### All Submissions:






### Changes proposed in this Pull Request:

* Moves the `usePreventLeavingPage` hook to the navigation package
* Renames the hook to `useConfirmUnsavedChanges`

Closes #36723 .






### How to test the changes in this Pull Request:



1. Navigate to the new product experience
2. Make some changes to the form
3. Click on a different menu item in the admin menu
4. Check that you are prompted to save before leaving



### Other information:




### FOR PR REVIEWER ONLY:
